packages/server/src/api/app/bsky/getRepostedBy.ts

```diff
@@ -48,7 +48,9 @@ export default function (server: Server) {
         encoding: 'application/json',
         body: {
           uri,
+          cid,
```

Comment:
This was added for the same reason as https://github.com/bluesky-social/adx/pull/242/files#r997401639